tasks/main.yml

```diff
@@ -0,0 +1,56 @@
+---
+
+###
+### Main entrypoint
+###
+
+- include: install-helm.yml
```

Comment:
Instead of installing helm on my machine (which I would not expect from this role), simply `assert` if there is a helm binary to be used.

Reply:
Then we would need to check for the version of Your Helm. For the sake of simplicity I suggest we attack it on another PR.

Reply:
Resolving as `install-helm.yml` was removed